Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: check-fix
+check-fix:
+	ruff check --fix --target-version py310
+
+.PHONY: check-format
+check-format:
+	ruff format
+
+.PHONY: check-ruff-all
+check-ruff-all:
+	ruff check --fix --target-version py310
+	ruff format
```

Comment:
This addition is related to https://github.com/tqec/tqec/pull/599#issuecomment-2950185959

Reply:
See my [last comment](https://github.com/tqec/tqec/pull/599#issuecomment-2955009016), this will likely not be needed anymore.